transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/ImmediateScheduler.java

```diff
@@ -43,17 +44,26 @@ public ImmediateScheduler(Executor executor, BackendRegistry backendRegistry) {
   }
 
   @Override
-  public void schedule(TransportContext transportContext, EventInternal event) {
```

Comment:
not completely related, but can you pls remove this class from the `main` source-set and put it into test, it's not used in the sdk, only in tests

Reply:
Done.